docs/azure_arc_jumpstart/azure_arc_k8s/eks/eks_terraform/_index.md

```diff
@@ -144,19 +121,31 @@ The following README will guide you on how to use the provided [Terraform](https
 
 * Run the ```terraform init``` command which will initialize Terraform, creating the state file to track our work:
 
-  ![Screenshot showing terraform init being run](./image6.png)
+  ![Screenshot showing terraform init being run](./terraform_init.png)
+
+* Plan the Terraform deployment by running the ```terraform plan -out=infra.out``` command.
+  Wait for the plan to finish:
+
+  ![Screenshot showing terraform apply being run](./terraform_plan.png)
```

Comment:
That masking is massy. Please use straight lines masking in screenshots

Reply:
New screenshot uploaded and context added for clarity.